runtime/src/accounts_db.rs

```diff
@@ -3487,35 +3487,20 @@ impl AccountsDb {
         AccountsHash::checked_cast_for_capitalization(balances.map(|b| b as u128).sum::<u128>())
     }
 
+    // remove this by inlining and remove extra unused params upto all callchain
```

Comment:
@jeffwashington FYI, this is possible further optimization opportunity for accounts hash calculation. I'm leaving this alone for now because I want to minimize this pr (this needs to be back-ported to v1.5)